pkg/inspector/image-inspector_test.go

```diff
@@ -143,7 +151,7 @@ func TestGetAuthConfigs(t *testing.T) {
 	}
 
 	for k, v := range tests {
-		ii := &defaultImageInspector{*v.opts, InspectorMetadata{}}
+		ii := &defaultImageInspector{*v.opts, iiapi.InspectorMetadata{}, &MockImageServer{}}
```

Comment:
can't you just pass nil here, why the mock?  I guess that means the next question is should we test the nil and non nil use case? :smile: